prow/crier/reporters/slack/reporter.go

```diff
@@ -75,11 +80,16 @@ func channel(prowCfg config.SlackReporter, jobCfg *v1.SlackReporterConfig) (stri
 	return host, channel
 }
 
-func reportTemplate(prowCfg config.SlackReporter, jobCfg *v1.SlackReporterConfig) string {
+func reportTemplate(prowCfg *config.SlackReporter, jobCfg *v1.SlackReporterConfig) string {
+	var res string
+	if prowCfg != nil {
+		res = prowCfg.ReportTemplate
+	}
+	// Prefer config in job
 	if jobCfg != nil && jobCfg.ReportTemplate != "" {
-		return jobCfg.ReportTemplate
+		res = jobCfg.ReportTemplate
 	}
-	return prowCfg.ReportTemplate
+	return res
```

Comment:
This also seems to hide errors due to missing config.

Reply:
ditto, not a bug introduced in this PR